beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -1,19 +1,34 @@
 package shared
 
 import (
-	"fmt"
-	"strconv"
-
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/pkg/errors"
-
-	fieldparams "github.com/prysmaticlabs/prysm/v4/config/fieldparams"
-	"github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
-	"github.com/prysmaticlabs/prysm/v4/consensus-types/validator"
-	eth "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
 )
 
+var errPayloadHeaderNotFound = errors.New("expected payload header not found")
```

Comment:
This error is only used in `beacon-chain/rpc/eth/shared/conversions_state.go` might be better to move it in there